### PR TITLE
Fix first option if disabled (#473)

### PR DESCRIPTION
**Pull request by markjoeljimenez**
### Description
Fix issue [#473](https://github.com/jshjohnson/Choices/issues/473).
The first disabled choice passed in as an option is not actually disabled.

### Reproduced example
https://jsfiddle.net/uf9bmpy6/

### Fix
We can simply pass in the disabled choice instead of checking whether it's preselected or not.

### CODE REVIEW

**Comment by jshjohnson** [comment]
Thanks for this @markjoeljimenez. Could you add some tests to cover this change please?

**Comment by markjoeljimenez** [comment]
Hey @jshjohnson, I haven't written tests before so please let me know if my tests are sufficient enough.

File: cypress/integration/select-one.spec.js [hunk 1]
```diff
@@ -861,5 +861,39 @@ describe('Choices - select one', () => {
...
     });
+
+    describe('disabled-first-by-options', () => {
+      beforeEach(() => {
+        cy.get('[data-test-hook=disabled-first-choice-by-options]')
+          .find('.choices')
+          .click();
+      });
+
+      it('first choice is disabled', () => {
```
Comment (jshjohnson) [inline cypress/integration/select-one.spec.js]:
```suggestion
      it('disables the first choice', () => {
```

File: cypress/integration/select-one.spec.js [hunk 2]
```diff
@@ -861,5 +861,39 @@ describe('Choices - select one', () => {
...
+        cy.get('[data-test-hook=disabled-first-choice-by-options]')
+        .find('.choices__list--dropdown .choices__list')
+        .children()
+        .first()
+        .should('have.class', 'choices__item--disabled');
+      });
+
+      let selectValue;
+  
+      it('first non-disabled choice is selected', () => {
```
Comment (jshjohnson) [inline cypress/integration/select-one.spec.js]:
```suggestion
      it('selects the first non-disabled choice', () => {
```

File: public/test/select-one.html [hunk 3]
```diff
@@ -230,6 +236,29 @@ <h2>Select one inputs</h2>
           removeItemButton: true,
         });
 
+        new Choices('#choices-disabled-first-choice-by-options', {
```
Comment (jshjohnson) [inline public/test/select-one.html]:
```suggestion
        new Choices('#choices-disabled-choice-via-options', {
```

**Comment by jshjohnson** [comment]
Hey - i've opened a new PR with your fix + some tweaks over latest master. Thanks 👍

**Pull request closed without merging**